### PR TITLE
New command: contract storage update

### PR DESCRIPTION
**Pull request by meevee98**
Also had to move `contract storage` to `contract storage get` because there are conflicts in the arguments with the new command

### CODE REVIEW

File: src/neoxp/Commands/ContractCommand.Storage.cs [hunk 1]
```diff
@@ -11,119 +11,237 @@
...
 using Newtonsoft.Json;
 using System.ComponentModel.DataAnnotations;
+using System.Numerics;
+using System.Text;
 
 namespace NeoExpress.Commands
 {
     partial class ContractCommand
     {
         [Command(Name = "storage", Description = "Display storage for specified contract")]
```
Comment (melanke) [inline src/neoxp/Commands/ContractCommand.Storage.cs:25]: I think this description needs to be updated

**Review by melanke — CHANGES_REQUESTED** [review]

File: src/neoxp/Commands/ContractCommand.Storage.cs (hunk 1 above)
Comment (melanke) [reply src/neoxp/Commands/ContractCommand.Storage.cs]: Resolved

**Review by melanke — APPROVED** [review]
Fine by me. But I am not an anthority to approve on this repo 😅

**Comment by lock9** [comment]
Hi @Jim8y ,

Can you check this, please? 

Thanks

**Review by Jim8y — APPROVED** [review]

**Pull request merged**